### ISSUE TITLE
"0" isn't rendered

### ISSUE THREAD

**Issue by egnikitin** [thephpleague/commonmark#376]
**Version(s) affected**: 0.19.2

**Description**  
If element contains "0" only it isn't rendered.  

**How to reproduce**  
`_0_`
`**0**`
print nothing.

**Possible Solution**  
src/HtmlElement.php
```
    public function __construct(string $tagName, array $attributes = [], $contents = '', bool $selfClosing = false)
    {
        $this->tagName = $tagName;
        $this->attributes = $attributes;
        $this->selfClosing = $selfClosing;

        $this->setContents($contents);
    }
```

```
 public function setContents($contents): self
    {
      $this->contents = $contents !== null ? $contents : '';
    }
```

**Comment by GrahamCampbell** [comment]
> Version(s) affected: 0.9.2

Do you mean 0.19.2 perhaps? ;)

**Comment by yenyasinn** [comment]
0.19.2 indeed

**Comment by colinodell** [comment]
Thanks for reporting this! I have released 0.19.3 with a fix.

**Comment by yenyasinn** [comment]
@colinodell looks like this fix doesn't work in php 7.1. Please check https://github.com/webuni/commonmark-table-extension/pull/23.

**Comment by colinodell** [comment]
It looks like you might be testing against version 0.19.0 instead - see my comment and screenshot on https://github.com/webuni/commonmark-table-extension/pull/23#issuecomment-508734817
